### ISSUE TITLE
LMMS Sharing Platform (LSP) content license issues (copyright, trademark)

### ISSUE THREAD

**Issue by qubodup** [LMMS/lmms.io#277]
Hello, LSP is great! Unfortunately it hosts some copyrighted content:
https://lmms.io/lsp/?action=show&file=15389 (megaman trademark/copyright)
https://lmms.io/lsp/?action=show&file=15394 (billie eilish copyright)

Some suggestions as food for thought:
1. Make by-nc-nd default, so that people who don't pay any attention to license at least don't upload a license that encourages commercial use/remixing - actions that might lead to legal trouble to people downloading from LSP and believing the license
2. Reduce the number of license options for new uploads while keeping old licenses (freesound did this to sampling+)
3. Add some kind of explanation(s) to the license choice section
4. Create a copyright report/moderation system
5. Delete copyrighted content(?)
6. Add a "fanart/unlicensed cover option (either as license or in addition to license - the latter is how blendswap does it)

The hope here is that these steps let LSP downloaders can more safely to remix without having to worry about infringing on 3rd party rights.

For reference, current license options:
![image](https://user-images.githubusercontent.com/18312/64881707-c95a0a00-d65b-11e9-8ee6-66048a453b04.png)

**Comment by liushuyu** [comment]
>Hello, LSP is great! Unfortunately it hosts some copyrighted content:
https://lmms.io/lsp/?action=show&file=15389 (megaman trademark/copyright)
https://lmms.io/lsp/?action=show&file=15394 (billie eilish copyright)
[edit] https://lmms.io/lsp/?action=show&file=15370 (galantis copyright)

This really subject to the jurisdiction as some jurisdictions may classify these under fair-use for their `transformative` work.

>Make by-nc-nd default, so that people who don't pay any attention to license at least don't upload a license that encourages commercial use/remixing - actions that might lead to legal trouble to people downloading from LSP and believing the license

Well `CC-BY-NC-ND` is much more strict than `CC-BY` and it __forbids__ remixing/deriving and commercial use of the licensed work.

>actions that might lead to legal trouble to people downloading from LSP and believing the license

I don't understand this statement. So are you saying that the people who uploaded their work does not actually agree they licensed their work under said license?

>Reduce the number of license options for new uploads while keeping old licenses (freesound did this to sampling+)

Probably. But LSP is not actively developed at this moment AFAIK so making someone to implement a change might be difficult.

>Create a copyright report/moderation system
Delete copyrighted content(?)


The system you are describing is a DMCA takedown system I think.

>Add a "fanart/unlicensed cover option (either as license or in addition to license - the latter is how blendswap does it)

I am not sure about this one, unlicensed is more problematic in some jurisdictions since these will be automatically recognized as "all rights reserved" work.

**Comment by Spekular** [comment]
@liushuyu regarding

> actions that might lead to legal trouble to people downloading from LSP and believing the license

My understanding of the proposal is that a more restrictive default license mitigates the effect of people uploading content they don't have the right to.

- I upload a remix of a song that I don't have the rights to
- I ignore the license dropdown because I'm not familiar with or don't care about licensing
- Scenario 1: The default license is "CC-BY". Anyone who comes across my remix thinks they can use ot howevee they want with accreditation. They could make commercial use of

**Comment by Spekular** [comment]
I hit "submit" too early, but I've completed my earlier comment now via edit.

**Comment by Umcaruje** [comment]
I don't think we should determine what is copyrighted and what is not, and delete content on our own. We should though state that we comply with the DMCA and maybe have a seperate email address for these infractions. If we get a DMCA takedown request for a file and it turns out to be copyrighted material, only then should we remove it.

**Comment by tresf** [comment]
Copyright (especially electronically, on mass) is a slippery slope.  All points made are valid, but what I consider to be most interesting is the idea of a user clicking the checkbox that says "I own this and have the right to upload" when it's a bit more gray.

In the case of these tracks, the slippery slope comes in the downstream licensee.  It's really not useful to claim the tracks uploaded to the LSP violate a copyright.  As @liushuyu has pointed out, they fall under fair use since they give accreditation and don't try to pass off someone else's work as their own.

The loophole is created by the licensee.  Anyone downloading this content can claim they're doing so under the license terms, so if a suit comes of it, they'll try to argue they were in compliance with the author's license.

Fortunately, judges are allowed to judge people and actions, and in the case of the listed tracks, each track is accredited by title and I'd be surprised to see a judge find either 1. Us (LMMS) or 2. Them (track authors) responsible for a violation.  The license provides weaker information than the track title.

So what do we do?  Well, the best is to fix the platform to embrace, but warn about this scenario.

* `by-nc-nd` - I feel strongly, no.  We're trying to move to CC0 to free our users of licensing restrictions, not encumber them.  I feel strongly that that's in the wrong direction.  Furthermore, masking ignorance with ignorance is just embracing some form of responsible licensing over these tracks.  `by-nc-nd` is just as inaccurate as "common public license".

I feel we should embrace better moderation of the platform and we're open to ideas as how to flag these tracks properly without changing the philosophical foundation of what sharing's about.

**Comment by tresf** [comment]
> Add a "fanart/unlicensed cover option (either as license or in addition to license - the latter is how blendswap does it)

👍 

> Add some kind of explanation(s) to the license choice section

👍 

> Reduce the number of license options for new uploads while keeping old licenses (freesound did this to sampling+)

👍 

> Create a copyright report/moderation system

👍 


I'd like to note, all these ideas involve editing the lsp codebase, of which is not well maintained.  We would be happy to accept PRs to enhance the platform.  It was written by the original author of LMMS many years ago and it's just a bunch of volunteers keeping it going.

**Comment by tresf** [comment]
Oh one more thing... I'd like to share a resource that specializes more in this type of problem:

https://ocremix.org/remix/OCR03971

There's no copyright info specifically, instead it says:

> **Primary Game:** Donkey Kong Country (Nintendo, 1994, SNES), music by David Wise, Eveline Novakovic, Robin Beanland
